webpack.config.js

```diff
@@ -23,9 +23,7 @@ const CssBlockRewriter = new CssBlocks.Rewriter(jsxCompilationOptions);
 const CssBlockAnalyzer = new CssBlocks.Analyzer(paths.appIndexJs, jsxCompilationOptions);
 
 const config = {
-  entry: {
-    main: './src/index.js',
-  },
+  entry: [ './src/index.jsx' ],
```

Comment:
Yeah, it is not supported, I had to change to `entry: { main: '' }` to make it work